modules/machineset-azure-trusted-launch.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+ifeval::["{context}" == "cpmso-using"]
+:cpmso:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="machineset-azure-trusted-launch_{context}"]
+= Configuring trusted launch for Azure virtual machines by using machine sets
+
+{product-title} {product-version} supports trusted launch for Azure virtual machines (VMs). By editing the machine set YAML file, you can configure the trusted launch options that a machine set uses for machines that it deploys. These machines can use UEFI security features such as Secure Boot or a dedicated virtual Trusted Platform Module (vTPM) instance.
```

Comment:
Similar thoughts to my previous comment: What's the logical connection between these two sentences. Is the second sentence an example of options that can be set? Or is it just additional info?